packages/snfoundry/contracts/src/YourCollectible.cairo

```diff
@@ -0,0 +1,221 @@
+use starknet::ContractAddress;
```

Comment:
I believe it looks like this because I started the migration using the base-challenge branch and added the necessary updates from challenge-0. However, the implementation is the same as what we currently have in challenge-0